src/gigs-board/components/molecule/markdown-editor.jsx

```diff
@@ -0,0 +1,30 @@
+const MarkdownEditor = ({
+  content,
+  onChange,
+}) => {
+  return (
+      <Widget
+        src="devhub.efiz.near/widget/SimpleMDE"
+        props={{
+          data: { content },
+          onChange,
+          toolbar: [
+            "heading",
+            "bold",
+            "italic",
+            "quote",
```

Comment:
Tried out every format hear. Majority of them looks great. The `quote` style, doesn't look obvious that it means to quote something:
<img width="350" alt="image" src="https://github.com/near/neardevhub-widgets/assets/13259400/cc07996f-3a96-4130-885e-2271855dd541">

Can we tweak the style a little bit?